packages/rust-bindings/src/aux_functions.rs

```diff
@@ -821,6 +822,76 @@ pub fn create_id_proof_aux(input: IdProofInput) -> Result<JsonString> {
     Ok(json!(out).to_string())
 }
 
+#[derive(SerdeDeserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Web3IdProofInput {
+    request: Request<constants::ArCurve, Web3IdAttribute>,
+    global_context: GlobalContext<constants::ArCurve>,
+    commitment_inputs: Vec<DeserializableCommitmentInputs>,
+}
+
+#[serde_with::serde_as]
+#[derive(SerdeDeserialize)]
+#[serde(rename_all = "camelCase", tag = "type")]
+pub enum DeserializableCommitmentInputs {
+    #[serde(rename_all = "camelCase")]
+    Account {
+        /// Issuance date of the credential that the proof is about.
+        /// This is an unfortunate name to conform to the standard, but the
+        /// meaning here really is `validFrom` for the credential.
+        issuance_date: String,
```

Comment:
We should be able to just use `chrono::DateTime<chrono::Utc>` here if we enable the `serde` feature of `chrono`.

Reply:
Yes, I missed that :thinking: